### PR TITLE
Verify MF user signup flow

### PR DESCRIPTION
**Pull request by abuzar-deriv**
1. Added MF user signup flow test
2. Moved assertion from c_addressDetails command to cypress/e2e/full/userSignupROW.cy.js to make the command reusable
3. Added new config variables

### CODE REVIEW

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
Looks really good. Just a minor issue regarding the use of ; (see coding standards in e2e wiki).
image

File: cypress/e2e/wip/Tradershub/accountSignupMF.cy.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  beforeEach(() => {
+    localStorage.setItem("config.server_url", Cypress.env("stdConfigServer"))
+    localStorage.setItem("config.app_id", Cypress.env("stdConfigAppId"))
+    cy.c_visitResponsive("/endpoint", "desktop")
+    cy.findByRole("button", { name: "Sign up" }).should("not.be.disabled")
+    cy.c_enterValidEmail(sign_up_mail)
+  })
+  it("Verify I can signup for an MF demo and real account", () => {
+    cy.wait(5000) //TODO - To be replaced by a loop within the emailVerification below.
```
Comment (rajendra-deriv) [inline cypress/e2e/wip/Tradershub/accountSignupMF.cy.js:20]: Do you have a card planned for it? Else plz keep track of this TODO

Comment (abuzar-deriv) [reply cypress/e2e/wip/Tradershub/accountSignupMF.cy.js:20]: I will update it within https://app.clickup.com/t/20696747/TRAH-2808

**Review by rajendra-deriv — APPROVED** [review]
Changes looks good to me. Approving.

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Pull request merged**